plugin.audio.tripler/LICENSE.txt

```diff
@@ -1,3 +1,6 @@
+This plugin is Copyright (c) 2022 Simon Mollema.
```

Comment:
If you use MIT license, please keep to the MIT format for it: https://choosealicense.com/licenses/mit/

Reply:
I have updated the license file to resolve this - thanks :)